securedrop_client/export.py

```diff
@@ -350,3 +365,6 @@ def print(self, filepaths: List[str]) -> None:
                 self.print_call_failure.emit(e)
 
         self.export_completed.emit(filepaths)
+
+
+Service = Export
```

Comment:
This is not something we've typically done in our code, but I understand why you did it and it does make the rest of the code (wherever you set up an `export.Service()`) readable.  No changes at this time - just commenting that we may want to coalesce around the same naming (/aliasing/importing/etc) strategies as a team

Reply:
> [...] just commenting that we may want to coalesce around the same naming (/aliasing/importing/etc) strategies as a team

Yes! (Naming is an important part of how I personally keep import cycles and more generally dependencies in check.)